### DISCUSSION TITLE
0.4.21 Nuget Release

### DISCUSSION THREAD

**Discussion by MattB-msft** [microsoft/PowerPlatform-DataverseServiceClient#157] [General]
Added optional WebAPI support during login and solution import & export operations
Refactored ServiceClient and ConnectionService
Added new dependency on System.Text.Json, preparing for the removal of Newtonsoft dependency

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/PowerPlatform-DataverseServiceClient/releases/tag/0.4.21'>0.4.21 Nuget Release</a>.</em>

**Comment by rajyraman** [comment]
@MattB-msft - Can we start using System.Text.Json in plugins and workflows? Is it possible?

**Reply by MattB-msft** [reply]
Not _yet_. 
Though we plan to have it available in the future, and allow your use of it before we have it as a standard lib. 
and Yes, that is intentionally vague as we have not yet announce a few things :) 

This change is in prep for removing Newtonsoft from the DV ServiceClient itself.
